content/en/Getting started/checklist.md

```diff
@@ -51,6 +51,36 @@ Under the Asset tab, you can review:
 {{% pentest-report-requirements %}}
 <!-- For content, see the following file: layouts/shortcodes/pentest-report-requirements.html -->
 
+### Co-branded Reports
+
+If your organization manages pentests for third parties, you can add your company logo to all reports. In co-branded reports, your company logo appears next to the Cobalt logo.
+
+An [_Organization Owner_](../glossary/#organization-owner) of a partner company can enable co-branded reports.
+
+{{%expand "Learn more." %}}
+
+<ol>
+<li>To turn on the co-branding functionality, submit a request to <a href="mailto:support@cobalt.io">support@cobalt.io</a> with:
+<ul><li>Your name</li>
+<li>Your organization name</li>
+<li>List of client organizations you manage</li>
+<li>Reason: Enable partner co-branding functionality</li></ul></li>
+
+<li>Once we've processed your request, you get access to specific functionality as a <i>Cobalt Partner</i>, which you can verify on the <b>People</b> tab next to your name.</li>
```

Comment:
@yogipetkar Updated. Please approve the PR if we are ready to go with this version.